regulations/static/regulations/js/source/views/comment/comment-view.js

```diff
@@ -56,18 +55,22 @@ var CommentView = Backbone.View.extend({
     });
 
     this.listenTo(CommentEvents, 'comment:target', this.target);
-    this.listenTo(CommentEvents, 'comment:update', this.update);
 
-    if (this.section) {
-      this.load();
-    }
+    this.setSection(options.section);
   },
 
-  render: function() {},
-
-  setSection: function(section) {
-    this.section = section;
-    this.key = 'comment:' + section;
+  setSection: function(section, overwrite) {
+    if (this.model) {
+      this.stopListening(this.model);
+    }
+    if (section) {
+      this.model = overwrite ?
+        new CommentModel({id: section}) :
+        comments.get(section) || new CommentModel({id: section});
+      this.listenTo(this.model, 'destroy', this.setSection.bind(this, section, true));
```

Comment:
When we initialize the `CommentView`, it doesn't have a model yet, and each time we call `setSection`, we get or create the appropriate `CommentModel`. The `overwrite` flag is a hack because I couldn't think of a better approach--on destroying a model, Backbone seems to call the destroy callback *before* removing the model from the collection, so that `comments.get` returns the pending-delete model when we get to this code. That's why I set the destroy callback on line 70 to pass `overwrite`. Any ideas for a cleaner approach?